indico/modules/events/timetable/templates/display/_common.html

```diff
@@ -0,0 +1,43 @@
+{% from 'events/notes/_note.html' import render_note %}
+{% from 'attachments/_display.html' import render_attachments_folders %}
+{% from 'events/display/_common.html' import render_users %}
+
+{% macro render_description(item) %}
+    <span itemprop="description" class="contrib-description html">
+        {{ item.description | safe }}
+    </span>
+{% endmacro %}
+
+{% macro render_references(item) -%}
+    {%- for reference in item.references: -%}
+        {% if reference.url -%}
+            <a href="{{ reference.url }}" target="_blank">{{ reference.urn }}</a>
```

Comment:
`reference.urn` may be `None` if a reference type does not have a scheme configured. `reference.urn or reference.value` as a fallback?